src/manage_sql_nvts.c

```diff
@@ -84,6 +89,31 @@ set_osp_vt_update_socket (const char *new_socket)
     }
 }
 
+/**
+ * @brief Get the number of extra days of VTs to fetch during feed update.
+ *
+ * @return The number of extra days of VTs to fetch during feed update.
+ */
+int
+get_vt_sync_extra_days ()
+{
+  return vt_sync_extra_days;
+}
+
+/**
+ * @brief Set the number of extra days of VTs to fetch during feed update.
+ *
+ * @param new_extra_days The new number of extra days of VTs to fetch.
+ */
+void
+set_vt_sync_extra_days (int new_extra_days)
+{
+  if (new_extra_days > 0)
+    vt_sync_extra_days = new_extra_days;
+  else
+    vt_sync_extra_days = 0;
```

Comment:
Could it be possible to set a different default like `7` here? Because otherwise at least community members will have the problem that if they don't know / notice the new `vt-sync-extra-days` parameter (which will definitely happen in most cases) will run into troubles sooner then later.

Another possibility would be to just use a default hardcoded "extra days" of something between 7 and 14 days (there should be no new/changed VT entering the feed with a last_modification date later then that) which would reduce the complexity of the code.